spec/i18n_spec.rb

```diff
@@ -80,27 +80,20 @@ class BaseTask
         { key: 'links.contact', locales: %i[fr] }, # "Contact" is "Contact" in French
         { key: 'saml_idp.auth.error.title', locales: %i[es] }, # "Error" is "Error" in Spanish
         { key: 'simple_form.no', locales: %i[es] }, # "No" is "No" in Spanish
+        { key: 'telephony.format_length.six', locales: %i[zh] }, # numeral is not translated
+        { key: 'telephony.format_length.ten', locales: %i[zh] }, # numeral is not translated
```

Comment:
These are numerals that we don't need to translate so I've moved them up into this section